mkdocs/docs/configuration.md

```diff
@@ -228,19 +228,19 @@ catalog:
 catalog:
   default:
     uri: thrift://localhost:9083
-    s3.endpoint: http://localhost:9000
```

Comment:
How about leaving those properties as well?

Reply:
Ok, I added back in https://github.com/apache/iceberg-python/pull/766/commits/f2b25227c3900a1920771bacb058e0ecc5442074